### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by niquola**

### CODE REVIEW

File: demo/public/example.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+resourceType: Patient
```
Comment (plynchnlm) [inline demo/public/example.yaml]: I think this came from patient-example.json.  Maybe keep the filename as patient-example.yaml, to keep the connection?

**Comment by plynchnlm** [comment]
It turns out that you can see the differences against master (after other branches are merged) if you change the target branch to something else, and then switch it back to master, which I what I just tried.

File: demo/public/app.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+const fhirpath = require('../../build/fhirpath.js');
+const codeMirror = require("codemirror");
+const yaml = require('js-yaml');
+
+require("codemirror/mode/yaml/yaml.js");
+require("codemirror/addon/lint/yaml-lint.js");
+require("codemirror/lib/codemirror.css");
+
+const example = require("json-loader!yaml-loader!./example.yaml");
```
[the remark this answers is not in the document]
Comment (niquola) [reply demo/public/app.js]: Yes

File: demo/public/example.yaml (hunk 1 above)
Comment (niquola) [reply demo/public/example.yaml]: Ok

**Pull request merged**